openslides/motions/static/js/motions/site.js

```diff
@@ -1048,6 +1053,382 @@ angular.module('OpenSlidesApp.motions.site', ['OpenSlidesApp.motions', 'OpenSlid
     }
 ])
 
+.factory('MotionDocxExport', [
```

Comment:
Please move this code to an extra file. See linenumbering,js and diff.js as an example. I won't check this quasi library. Later we have to exclude it from core.